exportlegends.lua

```diff
@@ -18,15 +18,30 @@ Options:
 :maps:   Exports all seventeen detailed maps
 :all:    Equivalent to calling all of the above, in that order
 
+Argument:
+:folder_name: (optional) The name of the folder where all the files will be saved.
```

Comment:
Turns out this doesn't render well (Sphinx doesn't warn about it, but it looks like this):
![image](https://user-images.githubusercontent.com/3719547/88507432-65f27580-cfaa-11ea-9802-c1fb4c43efbc.png)

I can fix it, but we also have a new guide up for building the docs, which I'd recommend for testing changes locally: https://docs.dfhack.org/en/latest/docs/Documentation.html

Reply:
Thanks for the link. Next time I add some documentation I'll take a look.